data/hai/hai reveal 4 middle.txt

```diff
@@ -1413,13 +1413,122 @@ mission "Hai Reveal [A11] Sound the Alarm"
+			`	There is a shift in the sound of the room and your attention is drawn to one Hai in particular, a particularly elderly man who struggles to his feet in a series of pained stages. The effect on the assembly is profound though. A ripple of silence flows outward from his position and elders all around promptly drop back into their seats as it reaches them, leaving a scant handful who were too slow to react to have their final words trail off awkwardly. Only the last speaker remains on her feet, the moderately weakly announcing what you know to be, "The council hears Aimet."`
+			`	"Security through secrecy..." He coughs. "You throw that in there as if it were always so Gemnali but, unlike you, I was there when it was added to the mandate of this chamber."`
+			`	He peers around the room and you realize he is blind, or nearly so. "Freedom may be achieved through peace, but peace must not be maintained at the cost of freedom. Order may not be pursued at the cost of peace, nor security at the cost of order. Always this mandate must be considered in both directions. To censure this woman, to resist action on this matter. You would have us pursue secrecy at the cost of security."`
+			`	He coughs again it echoes in the silence of nearly a hundred Hai holding their breath. When he continues his voice has grown stronger and more cutting. "My eyes may not see as well as they used to, but there are many in this chamber who are yet too blind to see. The edifice of our stability is not threatened by this woman who would have us restore security to our lost brethren. It is you who would use her virtue to topple it yourself."`
+			`	The silence stretches with all eyes on Gemnali. She clenches her fists, shaking with barely suppressed emotion, before collapsing back into her seat as if physically struck.`
+			`	Whatever spell had held the room is immediately broken and a susurrus of murmuring rises to fill its space. Within the noise a certain phrase starts to be repeated multiple times, drawing an expression of shock from Sayari.`
+			choice
+				`	"What are they saying?"`
+				`	"What just happened?"`
+			`	She glances at you briefly. "They are saying; Veeheylur and Misanaya fight over Eyrisen." Her tone carries with it a sense of awe and despite the growing volume in the room she leans in close to you and continues in scarcely more than a whisper. "The mythos has been invoked."`
+			`	As Aimet painfully resumes his seat, one Elder who has thus far been maintaining a fairly neutral demeanor in this seems to regain a certain sense of himself and stands up, quickly heralded by the moderator who struggles to compose himself suitably. "I think the council has heard enough. The petitioner is entitled to a closing statement unless there is objection?"`
```

Comment:
```suggestion
			`	As Aimet painfully resumes his seat, one Elder who has thus far been maintaining a fairly neutral demeanor in this seems to regain a certain sense of himself and stands up, quickly heralded by the moderator who is struggling to compose himself suitably. "I think the Council has heard enough. The petitioner is entitled to a closing statement, unless there is objection?"`
```
I'm no good at tenses, but this sounds better to me. I need an opinion from EjoThims.

Reply:
Both are correct, with very little distinction in meaning, so comes down to preference there.

Rest of these changes are necessary, though.

Reply:
```suggestion
			`	As Aimet painfully resumes his seat, one Elder who has thus far been maintaining a fairly neutral demeanor in this seems to regain a certain sense of himself and stands up, quickly heralded by the moderator who struggles to compose himself suitably. "I think the Council has heard enough. The petitioner is entitled to a closing statement, unless there is objection?"`
```